test-requirements.txt

```diff
@@ -15,10 +15,12 @@ sphinx-autobuild
 # nosetests enhancements
 rednose
 nose-timer
+# splitting tests run on a separate CI machines
+nose-parallel
 # Required by st2client tests
-pyyaml
+pyyaml<4.0,>=3.12
 RandomWords
-gunicorn>=19.3.0,<19.4
+gunicorn==19.6.0
```

Comment:
this is synced with main `requirements`, so we don't do pip install -> reinstall because the version diff.